crates/sui-cost/tests/snapshots/empirical_transaction_cost__good_snapshot.snap

```diff
@@ -4,55 +4,55 @@ expression: common_costs_actual
 ---
 {
   "MergeCoin": {
-    "computationCost": 1000,
+    "computationCost": 100000,
```

Comment:
@dariorussi / @oxade do you know why this PR could change the cost snapshot?

Reply:
if the gas price is 100 (which is what the PR claims) this is exactly the change you'd expect, no?

Reply:
resolved offline. This is the total usage, not unit